release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Solving bug with path to the vault temp directory [PAPP-32420]
```

Comment:
Can we change this to `Updating path to vault temp directory [PAPP-32420]` or to whatever the other apps use for updating the vault temp dir

Reply:
Done